main/build.gradle

```diff
@@ -62,6 +62,7 @@ repositories {
 dependencies {
     annotationProcessor project(':processor')
     compileOnly project(':processor')
+    compileOnly project(':comparator')
```

Comment:
I don't think you want the main CLI app to depend on comparator? Looks like the comparator module has it's own `Main` as an entry point.

Reply:
Indeed, removed :)